README.md

```diff
@@ -7,6 +7,118 @@ height="42" />
+                "url": "Hosting brand page url",
+                "hireExpertsInfo": "Marketplace service purchase url",
+                "support": "Hosting brand support contact url",
+                "adminPage": "Plugin Dashboard Homepage url",
+                "setup": {
+                    "payment": [
+                        Payment options supported for eg: "Paypal", "Razorpay", "Stripe"
+                    ],
+                    "shipping": [
+                        Shipping options supporter for eg: "Shippo"
+                    ]
+                },
+                "defaultContact": {
+                    "woocommerce_default_country": "Default country code",
+                    "woocommerce_currency": "Default currency code"
+                },
+                "wondercartBuyNow": "Marketplace link to purchase WonderCart"
+            },
+            "nonces": {
+                "gateway_toggle": ""gateway_toggle_code"
+            },
+            "install_token": "NFD_INSTALLER_token"
+        }
+    },
+    "siteUrl": "Your wordpress site url",
+    "siteTitle": "Your wordpress site name",
+    "restUrl": "Your wordpress site /wp-json/",
+    "restNonce": "restNonce_value",
+    "isWoocommerceActive": boolean,
+    "isYithBookingActive": boolean,
+    "isJetpackBoostActive": boolean,
+    "wpVersion": "Current WordPress version",
+    "currentTheme": "Current Theme Name",
+    "context": {
+        "platform": "default",
+        "brand": {
+            "name": "Hosting brand name"
+        }
+    },
+    "plugin": {
+            "url": "Brand plugin build url",
+            "version": "Brand plugin version",
+            "assets": "Brand plugin assets url",
+            "brand": "Current WordPress hosting brand name"
+    },
+    "comingSoon": {
+      enable: {
+        success: boolean
+      }  
+      isEnabled: boolean,
+      disabled: {
+        success: boolean
+      }
+      lastChanged: timeStamp when coming soon setting was updated last time
+      toggleAdminBarSiteStatus: null
+    }
+}
+```
+
+## Critical Paths
+
+* Runtime should provide WordPress site meta data.
+* Runtime module should also provide status of site capabilities like: 
+    1. If WordPress site has access to AI onboarding
+    2. If WordPress site has access to GlobalCTB
+    3. If WordPress site has access to HelpCenter
+    4. If WordPress site has Ecomdash plugin enabled
+    5. If WordPress site has any of Yith Extended plugins enabled 
+    6. If WordPress site is an Ecommerce store
+    7. If WordPress site is migrated to Jarvis
+* Runtime module should provide hosting brand settings information
+* Runtime module must give payment & shipping options available on Ecommerce store as per the geographic location.
+* Runtime module should give  current status information of `coming soon` mode.
+* Runtime module should support addition of custom values as per need under `NewfoldRuntime.sdk` variable.
```

Comment:
This isn't something we want the module to be able to do. We want to keep the data structure as flat as possible. Each module should register its own namespace (like `comingSoon`) and any other global items should be registered as part of the module.

Reply:
Updated content.